game/main.py

```diff
@@ -70,25 +70,20 @@
 level = levels[0]
 
 # Start the next level
-def next_level(level, levels, player):
+
+
+def next_level(level, levels):
```

Comment:
Extra spaces between comment and function should be removed
```suggestion
def next_level(level, levels):
```